src/mongoc/mongoc-version.h.in

```diff
@@ -91,5 +91,34 @@
          (MONGOC_MAJOR_VERSION == (major) && MONGOC_MINOR_VERSION == (minor) && \
           MONGOC_MICRO_VERSION >= (micro)))
 
+/**
+ * mongoc_get_major_version:
+ *
+ * Helper function to return the runtime minor version of the library.
```

Comment:
Copy/Paste from libbson again ;)
Seems, strange (imho, better to keep this in some installed file and available for users), but your CS, I will move them.